packages/@uppy/tus/src/index.js

```diff
@@ -154,7 +154,7 @@ module.exports = class Tus extends Plugin {
       }
 
       optsTus.onSuccess = () => {
-        this.uppy.emit('upload-success', file, upload, upload.url)
+        this.uppy.emit('upload-success', file, null, upload.url)
```

Comment:
we can drop the upload.urls here and instead pass an object that contains the `uploadURL` like in xhrupload